### PR TITLE
Testing/nl test edit details

### LINKED ISSUE #48: Write a reducers test. (state.js)
Start with state.js and actions.js?
- test GET_USER_DETAILS reducers
- test editDetails action

### PR DESCRIPTION
**Pull request by NicoleLatifi**
## Description
_What does this PR accomplish and what did you change/update/delete?_

Test for the reducer to return the initial state. Test for the reducer to return updated state when given new user details.

Also, we did some updates with uploading an image. We got rid of the getPhoto action and instead are using the editDetails action.

_Specify any bugs or details to come back to_

It is a known bug that when the user's profile page is refreshed, the updated image does not persist. But when the user logs out and then logs back in, it does persist.

## Closes issue(s)
resolves #48
makes progress on #53

### CODE REVIEW

File: code/package.json [hunk 1]
```diff
@@ -11,7 +11,8 @@
     "setup:mobile": "npm install --prefix mobile",
     "start": "concurrently 'npm run start:web' 'npm run start:api' --kill-others --names 'web,api'",
     "start:web": "npm run start --prefix web",
-    "start:api": "npm run start --prefix api"
+    "start:api": "npm run start --prefix api",
+    "test": "jest --watch"
```
Comment (ErinUntermeyer) [inline code/package.json:15]: Thanks for getting this installed properly so we can write tests!

File: code/web/src/modules/user/api/actions.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import * as actions from './actions';
+
+describe('actions', () => {
+  it.skip('should have a type of GET_USER_DETAILS', () => {
+    const user = { name: "ABC" }
+    const expectedAction = {
+      type: 'AUTH/GET_USER_DETAILS',
```
Comment (MarchCorbin) [inline code/web/src/modules/user/api/actions.test.js:7]: Awesome job getting the test set up and passing!

File: code/web/src/modules/user/api/state.test.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      "details": null,
+      "error": null,
+      "isAuthenticated": false,
+      "isLoading": false,
+    }
+
+    const user = { name: "Bob" }
+
+    const action = {
+      type: "AUTH/GET_USER_DETAILS",
```
Comment (ErinUntermeyer) [inline code/web/src/modules/user/api/state.test.js:32]: Awesome first two tests! I think we could add one more here for the editDetails method and then move on to testing the action creators.

**Review by ErinUntermeyer — APPROVED** [review]

**Review by MarchCorbin — APPROVED** [review]
This is masterful work getting the bugs squashed and setting up the tests!

**Pull request merged**